specification/schemas/components/MessageStatus.yaml

```diff
@@ -2,7 +2,10 @@ type: object
 title: CallbackRequest
 properties:
   type:
-    const: MessageStatus
```

Comment:
`const` is not a valid OpenAPI schema keyword, so I've replaced it with a string that has a single value to replicate the same behaviour